src/main/java/org/molgenis/vcf/decisiontree/loader/model/ConfigBoolClause.java

```diff
@@ -0,0 +1,76 @@
+package org.molgenis.vcf.decisiontree.loader.model;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+public class ConfigBoolClause {
```

Comment:
why isn't this a AutoValue class similar to `ConfigBoolQuery`?

Reply:
The combination of autovalue and the constructors needed for deserialisation are not possible here due to the auto ID